community/kernel/src/test/java/org/neo4j/test/subprocess/SubProcess.java

```diff
@@ -214,7 +197,7 @@ private String classPath( String parentClasspath )
 
     private static Process start( String... args )
     {
-        ProcessBuilder builder = new ProcessBuilder( args );
+        ProcessBuilder builder = new ProcessBuilder( args ).inheritIO();
```

Comment:
@chrisvest i have tried to run ha and recovery robustness locally and those are green (did not check if we missing some output that's true) 
@thobe will propose to try to merge it first and in case if it will cause some tests failures add those into a comments (to not face same question again) also in that case will follow your suggestion: will add possibility to inherit IO and will keep current piping. 

Just to have it here. That somehow may be related to broken io inheritance:
http://bugs.java.com/view_bug.do?bug_id=8023130
https://bugs.openjdk.java.net/browse/JDK-8023130